### PR TITLE
[Refactor] Modal

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: Yash Kamboj <yashkamboj29@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/Dialog.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import {Box, Dialog, DialogTitle, DialogActions, Backdrop, Fade } from '@mui/material'
```
Comment (Chadha93) [inline ui/components/Dialog.js:1]:
```suggestion
import {Box, Dialog, DialogTitle, DialogActions, Backdrop, Fade } from '@mui/material';
```

**Review by Chadha93 — COMMENTED** [review]
Looks good @YashKamboj, a question regarding themes and color reference.

File: ui/styles/theme.js [hunk 2]
```diff
@@ -53,6 +53,13 @@ const theme = createTheme(
       // MuiIconButton: { root: { padding: theme.spacing(1) } },
       MuiTooltip: { tooltip: { borderRadius: 4 } },
       MuiDivider: { root: { backgroundColor: "#404854" } },
+      MuiDialogTitle: { 
+        root: {
+          background: '#607d8b',
```
[the remark this answers is not in the document]
Comment (YashKamboj) [reply ui/styles/theme.js:58]: @Chadha93 I will make a doc explaining where we can use the theme creator

**Comment by YashKamboj** [comment]
@Chadha93 Is this good to merge??

**Comment by Chadha93** [comment]
> @Chadha93 Is this good to merge??, unable to resolve the dco issue

Yes, getting you covered on DCO for this.

**Comment by YashKamboj** [comment]
Thank you!!

**Pull request merged**